app/services/NextUpdatesService.scala

```diff
@@ -98,12 +98,12 @@ class NextUpdatesService @Inject()(val obligationsConnector: ObligationsConnecto
 
   }
 
-  def getSubmissionCounts(optOutTaxYear: TaxYear)
-                         (implicit hc: HeaderCarrier, mtdUser: MtdItUser[_]): Future[SubmissionsCountForTaxYear]  = {
+  def getQuarterlyUpdatesCounts(optOutTaxYear: TaxYear)
```

Comment:
should fix: `optOutTaxYear` doesn't give any context of `NextUpdatesService`, could rename it to `taxYear`

Reply:
done